internal/backend/factory/factory.go

```diff
@@ -81,29 +68,29 @@ func (handler *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	params, err := parseRequest(r, handler.state, handler.config)
-	if err != nil {
-		if errors.Is(err, errNotFound) {
-			httpNotFound(w)
+	path := r.URL.Path
 
-			return
-		}
-
-		handler.handleError("failed to parse request", w, err)
+	proxyURL, err := getProxyURL(handler.config.ImageFactoryBaseURL, path)
```

Comment:
This change is to forward all requests to image factory as-is (we have the same url format) which gives support for the non-iso URLs like `1.11.5/metal-arm64.raw.zst`